scimono-server/src/main/java/com/sap/scimono/entity/validation/patch/PatchAttributeMutabilityValidator.java

```diff
@@ -1,35 +1,34 @@
 
 package com.sap.scimono.entity.validation.patch;
 
-import com.sap.scimono.entity.patch.PatchOperation.Type;
 import com.sap.scimono.entity.schema.Attribute;
 import com.sap.scimono.entity.validation.Validator;
 import com.sap.scimono.exception.SCIMException;
 
 public class PatchAttributeMutabilityValidator implements Validator<Attribute> {
 
-  private Type operationType;
+  private boolean isOperationReplacing;
 
-  public PatchAttributeMutabilityValidator(Type operationType) {
-    this.operationType = operationType;
+  public PatchAttributeMutabilityValidator(final boolean isOperationReplacing) {
```

Comment:
I made it a boolean value because "add" operation under certain conditions might replace and "replace" under certain conditions might add. The problem is there is no way to tell without getting the resource, so this is why I haven't changed the behaviour. Should I return it to operation type?